packages/frontend/src/components/UserSettings/UsersAndGroupsPanel/UsersView.tsx

```diff
@@ -266,184 +259,88 @@ const UserListItem: FC<{
     );
 };
 
-const UserManagementPanel: FC = () => {
-    // TODO: this is a feature flag while we are building groups.
-    // Remove this when groups are ready to be released.
-    const groupManagementEnabled = useFeatureFlagEnabled('group-management');
-
-    const { classes } = useTableStyles();
-    const { user } = useApp();
+const UsersView: FC = () => {
     const [showInviteModal, setShowInviteModal] = useState(false);
-    const [showCreateGroupModal, setShowCreateGroupModal] = useState(false);
+    const { user } = useApp();
+    const { classes } = useTableStyles();
 
     const [search, setSearch] = useState('');
+
     const { data: organizationUsers, isLoading: isLoadingUsers } =
         useOrganizationUsers(search);
-    const { data: groups, isLoading: isLoadingGroups } =
-        useOrganizationGroups();
 
-    if (user.data?.ability.cannot('view', 'OrganizationMemberProfile')) {
-        return <ForbiddenPanel />;
+    if (isLoadingUsers) {
```

Comment:
with the removal of this check: `user.data?.ability.cannot('view', 'OrganizationMemberProfile')` do we need to add it somewhere else?

Reply:
This is a silly artifact of the bad diff. This now happens in the parent file.